src/UniqueCharacters.java

```diff
@@ -4,6 +4,7 @@ public static void main(String[] args) {
     }
 
     public static boolean uniqueCharacters(String input) {
+        input = input.replaceAll("\\s+", "");
```

Comment:
create a new variable to avoid receive the same variable that you are changing.

Reply:
Done.